### PR TITLE
move share config from global to user

### PR DESCRIPTION
**Pull request by ZeddZull**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Documentation | no
| Translation   | no
| Fixed tickets | #1779
| License       | MIT

The migration class is not ready yet, we need to define how the update will be done first.
Also it isn't available for baggy theme.

### CODE REVIEW

File: src/Wallabag/CoreBundle/Resources/views/themes/material/Config/index.html.twig [hunk 1]
```diff
@@ -28,14 +28,6 @@
...
                             <div class="row">
                                 <div class="input-field col s12">
-                                    {{ form_errors(form.config.theme) }}
-                                    {{ form_widget(form.config.theme) }}
-                                    <label class="required">{% trans %}Theme{% endtrans %}</label>
-                                </div>
-                            </div>
-
-                            <div class="row">
-                                <div class="input-field col s12">
```
Comment (j0k3r) [inline src/Wallabag/CoreBundle/Resources/views/themes/material/Config/index.html.twig]: Why did you remove these lines?

Comment (ZeddZull) [inline src/Wallabag/CoreBundle/Resources/views/themes/material/Config/index.html.twig]: I fixed it, just forgot to put it back after some experiments .

File: src/Wallabag/CoreBundle/Command/InstallCommand.php [hunk 2]
```diff
@@ -223,6 +223,14 @@ protected function setupAdmin()
         $config->setItemsPerPage($this->getContainer()->getParameter('wallabag_core.items_on_page'));
         $config->setRssLimit($this->getContainer()->getParameter('wallabag_core.rss_limit'));
         $config->setLanguage($this->getContainer()->getParameter('wallabag_core.language'));
+        $config->setCarrot(false);
+        $config->setShareDiaspora(false);
+        $config->setDiasporaUrl('http://diasporapod.com');
+        $config->setShareShaarli(false);
+        $config->setShaarliUrl('http://myshaarli.com');
+        $config->setShareMail(true);
+        $config->setShareTwitter(true);
```
Comment (j0k3r) [inline src/Wallabag/CoreBundle/Command/InstallCommand.php]: Instead of `Share*`, I would rather write them `Enable*`  instead. What do you think?

Comment (ZeddZull) [inline src/Wallabag/CoreBundle/Command/InstallCommand.php]: Yes good idea, i'll do that.

**Comment by j0k3r** [comment]
@ZeddZull any progress on this ?

**Comment by tcitworld** [comment]
The PR was closed because we renamed branch `v2` to `master`. Please send your PR again.
